public/enhanced-table-vis-params.html

```diff
@@ -190,8 +190,14 @@
 
     <div class="euiSpacer euiSpacer--s"></div>
     <div class="form-group">
-      <label>Hidden columns</label>
-      <input type="text" ng-model="editorState.params.hiddenColumns" class="form-control" placeholder="0,1,2,column_label_X..." />
+      <label>
+        Hidden columns
+        <icon-tip
+          content="'A column can be referenced by its index (e.g. 1,2,3..) or by its label (e.g. example_column,Example Column). It is important not to put any superscript or inverted comma if they are not part of the column label and to separate the columns to hide simply using a comma (labels containing commas cannot be hidden). References by id and by label can be mixed (e.g. 1,2,ExampleColumn) but it is strongly recommended to use the reference by label feature since indices may vary according to the timerange set.'"
+          position="'right'"
+        ></icon-tip>
+      </label>
+      <input type="text" ng-model="editorState.params.hiddenColumns" class="form-control" placeholder="0,1,2,Col2 Label,..." />
```

Comment:
As you set `Col2 Label`, can you remove `2` index?

Reply:
Sure. I missed that one. Thank you